src/lib/rest/rest.cpp

```diff
@@ -1168,6 +1182,8 @@ static int restStart(IpVersion ipVersion, const char* httpsKey = NULL, const cha
                                    MHD_OPTION_HTTPS_MEM_KEY,            httpsKey,
                                    MHD_OPTION_HTTPS_MEM_CERT,           httpsCertificate,
                                    MHD_OPTION_CONNECTION_MEMORY_LIMIT,  memoryLimit,
+                                   MHD_OPTION_CONNECTION_LIMIT,         maxConns,
+                                   MHD_OPTION_THREAD_POOL_SIZE,         threadPoolSize,
```

Comment:
What is the effect of the MHD_OPTION_THREAD_POOL_SIZE flag when MHD_USE_THREAD_PER_CONNECTION is used? Is it ignored?